index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="#" class="toggle-button">
```

Comment:
Good job working on the mobile first 👍 

- [ ] Kindly make sure your website looks exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=0ECBEqbggTjVdj0j-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃 

-  the `background image` link is broken.
- the `icons` link is broken.
- kindly, make sure you added this button
 ![btn](https://user-images.githubusercontent.com/57408419/227179624-54a4c69e-77aa-4a0e-9331-b0c0f4a5a5fa.JPG)
- Also make sure you fixed the color of the `menu` button 
![menu](https://user-images.githubusercontent.com/57408419/227180294-37e0577f-1297-4ac8-947f-7357a8f72199.JPG)




> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![mobile01](https://user-images.githubusercontent.com/57408419/227177710-a6bde6c9-b57c-4513-b83e-bf0da21ab2b2.JPG)| |![mob10](https://user-images.githubusercontent.com/57408419/227199707-bdc72733-0cd0-4dc6-b3d6-29eaf0797e43.jpg)| |